components/docs/docs.css

```diff
@@ -0,0 +1,102 @@
+/* navbar  */
+.docs {
+  background-color: var(--secondary);
+}
+/* sidebar  */
+.components-container {
+  display: grid;
+  /* grid-auto-flow: column; */
+  grid-template-columns: 1fr 5fr;
```

Comment:
Try to Remove commented code If not needed

Reply:
Everything Else Looks Good

Reply:
> Try to Remove commented code If not needed

Thanks, Will do that.